tests/integration_tests/performance/test_network_ab.py

```diff
@@ -27,7 +27,12 @@ def consume_ping_output(ping_putput, request_per_round):
     assert len(output) > 2
 
     # Compute percentiles.
-    seqs = output[1 : request_per_round + 1]
+
+    pattern_received = ".+ packets transmitted, (.+) received, .*"
+    received = int(re.findall(pattern_received, output[-2]))
+    assert request_per_round - 1 <= received, "Lost more than 1 packet during ping"
+
+    seqs = output[1 : received + 1]
```

Comment:
Why is 1 okay? Firecracker's net device should be a lossless channel, so we should never observe any packet loss :thinking:

Reply:
removed this for now